keras/src/backend/tensorflow/numpy.py

```diff
@@ -1855,6 +1855,26 @@ def ravel(x):
     return tf.reshape(x, [-1])
 
 
+def unravel_index(x, shape):
```

Comment:
Please add a check that `shape` does not contain `None` entries.

Reply:
Done